pippy/IR.py

```diff
@@ -1117,6 +1138,49 @@ def __str__(self):
     def __repr__(self):
         return self.split_gm.__repr__()
 
+    @staticmethod
+    def _get_param_buffer_mapping(
```

Comment:
Maybe you can leave a TODO saying to delete this when 2.3 release comes around with https://github.com/pytorch/pytorch/pull/115462?

Reply:
Good suggestion! Done.